spikes/npgsql-vanilla/README.rst

```diff
@@ -0,0 +1,40 @@
+#################################################################
+Basic demonstration program for using CrateDB with vanilla Npgsql
+#################################################################
+
+
+************
```

Comment:
because this is a short document with only one substantial section ("Usage") it is probably best to drop this heading

we try to include a few intro paras right after the title in most of our RST documents. "Introduction" sections are then only needed if there is a need to go into more detail

Reply:
Addressed with ca0e6edc. Thanks!